src/nodes/AssemblyStackAssignment.js

```diff
@@ -1,7 +1,12 @@
+import { doc } from 'prettier';
+
+const { hardline } = doc.builders;
+
 export const AssemblyStackAssignment = {
   print: ({ node, path, print }) => [
     path.call(print, 'expression'),
-    ' =: ',
+    hardline,
```

Comment:
In Slang (and also in the Solidity documentation)
The stack assignment starts at the `=:` and it's a statement by itself.
to pair it with the previous statement, would cause a very particular exception in the slang implementation which I believe is not very productive, specially for a feature that was removed on version `0.5.0`.